package.json

```diff
@@ -16,39 +16,6 @@
     "last 2 iOS versions",
     "last 2 Safari versions"
   ],
-  "babel": {
-    "presets": [
-      [
-        "es2015",
-        {
-          "modules": false
-        }
-      ],
-      "react"
-    ],
-    "env": {
-      "test": {
-        "presets": [
-          "es2015",
-          "react"
-        ]
-      },
-      "production": {
-        "plugins": [
-          [
-            "transform-react-remove-prop-types",
```

Comment:
For future reference, this still seemed to work as expected, and saved about 3.5KB of JS on the production build. I decided not to add it back since we would want to replace PropTypes with TypeScript type annotations anyway.